src/routers/AppRouter.js

```diff
@@ -51,6 +51,7 @@ const AppRouter = () => {
         {/* <PublicRoute exact path="/confirm-email/:confirmationId" component={ConfirmPage} /> */}
 
         <PrivateRoute exact path="/account" component={ProfilePage} />
+        <PrivateRoute exact path="/playlist/preview/:playlistId/resource/:resourceId" component={PreviewPage} previewType="playlist" />
```

Comment:
Make it multiline as like other route

Reply:
Hmm... why is that moved from bottom to top? Any issue with previous?

Reply:
fixed. these route was commented out before.

Reply:
it as previously in block of commented code. we have removed it from there and paster above